servers/realmi/metadata.json

```diff
@@ -0,0 +1,33 @@
+{
+    "id": "realmi",
+    "name": "Realmi",
+    "description": "Suomalainen Peliyhteisö. Liity mukaan jatkuvasti kasvavaan yhteisöömme!",
+    "addresses": [
+        "realmi.fi"
+    ],
+    "primaryAddress": "realmi.fi",
+    "minecraftVersions": [
+	"1.8.*",
+	"1.19.*"
+    ],
+    "primaryMinecraftVersion": "1.19.2",
+    "primaryColor": "#fb6900",
+    "secondaryColor": "#ffa800",
+    "primaryLanguage": "fi",
```

Comment:
we do not support this language. please review the README.MD for what is supported.

Reply:
Primary language has been changed